src/app/features/collection/components/chip-list/chip-list.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="chip-list">
+  @for (chip of chipList(); track chip) {
```

Comment:
we might want to track by some more reliable value, not object reference. I'm not sure we will keep reference the same